includes/class-freemius.php

```diff
@@ -7544,6 +7545,45 @@ function _activate_plugin_event_hook() {
             $this->_storage->was_plugin_loaded = true;
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.8.0
+         *
+         * @param string $plugin
+         * @param bool   $network_wide
+         */
+        function _activated_plugin( $plugin, $network_wide ) {

```

Comment:
Proposing to rename this function to `_handle_free_and_premium_parallel_activation` or something like that, just for readability.